sdk/web-pubsub/web-pubsub-express/review/web-pubsub-express.api.md

```diff
@@ -60,6 +62,87 @@ export interface DisconnectedRequest {
     reason?: string;
 }
 
+// @public
+export interface MqttConnectEventErrorResponse {
+    mqtt: MqttConnectEventErrorResponseProperties;
+}
+
+// @public
+export interface MqttConnectEventErrorResponseProperties {
+    code: MqttV311ConnectReturnCode | MqttV500ConnectReasonCode;
+    reason?: string;
+    userProperties?: MqttUserProperty[];
+}
+
+// @public
+export interface MqttConnectionContext extends ConnectionContext {
+    physicalConnectionId: string;
+    sessionId?: string;
+}
+
+// @public
+export interface MqttConnectProperties {
```

Comment:
All the fields except protocol version are nullable.

Reply:
For the nullability of these fields, check https://github.com/Azure/azure-webpubsub/blob/main/protocols/server/cloud-events/main.tsp